docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -195,6 +195,8 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 - `DISABLE_GIT_HOOKS`: **false**: Set to `true` to prevent all users (including admin) from creating custom
    git hooks.
 - `IMPORT_LOCAL_PATHS`: **false**: Set to `false` to prevent all users (including admin) from importing local path on server.
+- `INTERNAL_TOKEN`: **<empty>**: Secret used to validate communication within Gitea binary.
+- `INTERNAL_TOKEN_URI`: **<empty>**: Instead of defining internal token in the configuration, this configuration option can be used to give gitea a path to a file that contains the internal token (example value: `file:/etc/gitea/internal_token`)
```

Comment:
`this configuration option can be used to give gitea...` 
Uppercase "Gitea" here as well for consistency, a small nit and otherwise non-blocking.

Reply:
```suggestion
- `INTERNAL_TOKEN_URI`: **<empty>**: Instead of defining internal token in the configuration, this configuration option can be used to give Gitea a path to a file that contains the internal token (example value: `file:/etc/gitea/internal_token`)
```